src/classes/STG_InstallScript.cls

```diff
@@ -228,6 +233,33 @@ global without sharing class STG_InstallScript implements InstallHandler {
         upsert orgSettings;
     }
 
+    /*******************************************************************************************************
+    * @description Enabled Auto Population of Affiliation on Student Course Connection.
+    * @return void
+    */
+    global static void enableAutoPopulationOfAffilOnStudentCourseCxn() {
+        Hierarchy_Settings__c orgSettings = UTIL_CustomSettingsFacade.getSettings();
+        if (orgSettings.Populate_Affil_on_Student_Couse_Cxn__c == null || orgSettings.Populate_Affil_on_Student_Couse_Cxn__c != true) {
+            orgSettings.Populate_Affil_on_Student_Couse_Cxn__c = true;
+        }
+        upsert orgSettings;
+    }
+
+    /*******************************************************************************************************
+    * @description Disable Auto Population of Affiliation on Student Course Connection for push upgrade.
+    * @return void
+    */
+    global static void disableAutoPopulationOfAffilOnStudentCourseCxn(Version previousVersion) {
+        if(previousVersion != null && previousVersion.compareTo(new Version(1,92)) < 0) {
```

Comment:
This is current set to 1.92 but it will potentially changed based on what release this WI is assigned to.

Reply:
I wanna make sure I understand how this work, (sorry my brain is vacationing today).
Given existing customer is on 1.91, and getting upgraded to 1.92, we check to see if the connection is null or perhaps they've turned it on through Hierarchy Settings (like we told them not to), we set this to false.
Next release they go up to 1.92 and this block of code no longer runs so we aren't accidentially turning anyone off if perhaps they've turned it on after getting 1.92, correct?